src/main/menus/app.js

```diff
@@ -136,6 +136,13 @@ var createTemplate = function(mainWindow, config) {
       click: () => {
         mainWindow.webContents.send('zoom-in', -1);
       }
+    }, {
+      label: 'Zoom Out (hidden)',
+      accelerator: 'CmdOrCtrl+_',
+      visible: false,
+      click: () => {
```

Comment:
@kytwb Sorry, I had misunderstood your question. You had to rebase in order to track the latest master branch. For now, please rebase to resolve conflicts.